package.json

```diff
@@ -24,6 +24,7 @@
   "license": "ISC",
   "workspaces": [
     "enigma-coinjoin-operator",
-    "enigma-coinjoin-client"
+    "enigma-coinjoin-client",
+    "app"
```

Comment:
Small detail, but can you please use the same naming convention `enigma-coinjoin-app`?